include/native/windows/oaidl.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+// Don't care.
```

Comment:
Any reason why we need these empty headers?

Reply:
The DX headers include them for common IDL stuff but they don't define anything useful or needed.